lib/stacks/eks-blueprint-stack.ts

```diff
@@ -42,7 +42,7 @@ export class EksBlueprintProps {
     /**
      * Kubernetes version (must be initialized for addons to work properly)
      */
-    readonly version?: KubernetesVersion = KubernetesVersion.V1_25;
+    readonly version?: KubernetesVersion | "auto" = KubernetesVersion.V1_25;
```

Comment:
Let's remove this assignment to 1.25 - 1/ the auto is setting to 1.27 and 2/ we don't want more than one place where defaults will apply.